includes/admin/class-wc-stripe-admin-notices.php

```
@@ -680,13 +680,10 @@ public function subscription_check_detachment() {
-		$customer_stripe_page = sprintf(
-			'<a href="%s">%s</a>',
-			esc_url( WC_Stripe_Subscriptions_Helper::STRIPE_CUSTOMER_PAGE_BASE_URL . WC_Stripe_Order_Helper::get_instance()->get_stripe_customer_id( $subscription ) ),
-			esc_html(
-				/* translators: this is a text for a link pointing to the customer's page on Stripe */
-				__( 'Stripe customer page &rarr;', 'woocommerce-gateway-stripe' )
-			)
+		$customer_stripe_page = WC_Stripe_Helper::get_external_link(
+			WC_Stripe_Subscriptions_Helper::STRIPE_CUSTOMER_PAGE_BASE_URL . WC_Stripe_Order_Helper::get_instance()->get_stripe_customer_id( $subscription ),
+			/* translators: this is a text for a link pointing to the customer's page on Stripe */
+			__( 'Stripe customer page &rarr;', 'woocommerce-gateway-stripe' )
 		);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'subscription_check_detachment'. (undefined)

(StaticAccess)